dev/.rat-excludes

```diff
@@ -139,3 +139,4 @@ core/src/main/resources/org/apache/spark/ui/static/package.json
 testCommitLog
 .*\.har
 .nojekyll
+dummy_data.custom
```

Comment:
FYI, if you use `.csv` or `.data` extension for this new file, we don't need to touch this file.

Reply:
Thanks! I've changed it to reuse `.data`